core/controllers/question_test.py

```diff
@@ -111,3 +111,350 @@ def test_get(self):
             '%s/batch' % feconf.QUESTION_DATA_URL, self.payload,
             expect_errors=True)
         self.assertEqual(response.status_int, 404)
+
+
+class QuestionsHandlerTest(test_utils.GenericTestBase):
```

Comment:
All the docstrings for the newly added test classes are "Test the question handler" or variants thereof. This does not help to distinguish the test classes. Could you please rename the test classes to indicate which parts they're testing and update the docstrings to describe clearly the difference between the different classes?

Reply:
Updated.